content/chapters/compute/lab/solution/sleepy/sleepy_creator.py

```diff
@@ -1,19 +1,19 @@
 import subprocess
 from sys import exit
-from time import sleep
 
 NUM_SLEEPS = 10
 
 
 def main():
     # TODO 1: create 10 `sleep 1000` processes using `subprocess.Popen`
     # Use the documentation: https://docs.python.org/3/library/subprocess.html#subprocess.Popen
+    procs = []
     for _ in range(NUM_SLEEPS):
-        subprocess.Popen(["sleep", "1000"])
-
-    # TODO 2: Make this script also wait for 1000 seconds.
-    sleep(1000)
+        procs.append(subprocess.Popen(["sleep", "1000"]))
```

Comment:
```suggestion
        # Create new process and add it to the list of processes.
        p = subprocess.Popen(["sleep", "1000"])
        procs.append(p)
```
It makes things more readable. We are more concerned with readability then with conciseness.